homeassistant/components/simplefin/sensor.py

```diff
@@ -40,6 +41,12 @@ class SimpleFinSensorEntityDescription(SensorEntityDescription):
         unit_fn=lambda account: account.currency,
         icon_fn=lambda account: account.inferred_account_type,
     ),
+    SimpleFinSensorEntityDescription(
```

Comment:
Perhaps this sensor should be disabled by default and likely also be put in diagnostics section of the device

Reply:
I'll go diagnostic I guess